src/cbmc/cbmc_parse_options.cpp

```diff
@@ -750,7 +752,8 @@ int cbmc_parse_optionst::get_goto_program(
     return CPROVER_EXIT_SUCCESS;
   }
 
-  if(cbmc_parse_optionst::process_goto_program(goto_model, options, log))
+  if(cbmc_parse_optionst::process_goto_program(
+       goto_model, options, log, cmdline))
```

Comment:
Couldn't you just copy the `cmdline` part that you need into `options` instead of passing the whole thing?